app/views/cooperatives/registrations/new.html.erb

```diff
@@ -9,20 +9,25 @@
   </div>
 
   <div class="field">
-    <%= f.label :password %>
+    <%= f.label :password, 'Senha' %>
```

Comment:
Poderiamos manter todos textos no i18n. Aqui basta seguir o modelo de tradução de models da documentação: https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models